pkg/schedule/schedulers/hot_region.go

```diff
@@ -1225,12 +1228,12 @@ func (bs *balanceSolver) isUniformSecondPriority(store *statistics.StoreLoadDeta
 
 // calcProgressiveRank calculates `bs.cur.progressiveRank`.
 // See the comments of `solution.progressiveRank` for more about progressive rank.
-// | ↓ firstPriority \ secondPriority → | isBetter | isNotWorsened | Worsened |
-// |   isBetter                         | -4       | -3            | -1 / 0   |
-// |   isNotWorsened                    | -2       | 1             | 1        |
-// |   Worsened                         | 0        | 1             | 1        |
+// | ↓ firstPriority \ secondPriority → | isBetter | isNotWorsened | Worsened  |
+// |   isBetter                         | 4        | 3             | 1         |
+// |   isNotWorsened                    | 2        | -1            | -1        |
+// |   Worsened                         | 0        | -1            | -1        |
 func (bs *balanceSolver) calcProgressiveRankV1() {
```

Comment:
How about using variables to represent instead of magic numbers?

Reply:
It is the score to use comparing.

Reply:
Yes, it is. But it is still being used in many places for making judgments, which indeed can be hard to understand in the code, so more comments have to be added to explain it. Like these

https://github.com/tikv/pd/blob/cbba49ab32ddd82d86194abd8cf9c25d1db0c667/pkg/schedule/schedulers/hot_region.go#L1373-L1393

I feel like it's really hard to name, so I'm just suggesting something and not rushing to solve it. :)